frontend/Makefile

```diff
@@ -18,7 +18,7 @@ lint:
 
 .PHONY: mypy
 mypy:
-	mypy --ignore-missing-imports --follow-imports=skip --strict-optional --warn-no-return .
+	mypy --ignore-missing-imports --follow-imports=skip --strict-optional --warn-no-return --exclude venv .
```

Comment:
I would move mypy config to setup.cfg instead of complicating the call here:

https://mypy.readthedocs.io/en/stable/config_file.html#confval-exclude

wdyt?

Reply:
agreed, will move that over in the next commit